pootle/apps/pootle_store/models.py

```diff
@@ -1437,8 +1439,11 @@ def delete(self, *args, **kwargs):
         super(Store, self).delete(*args, **kwargs)
 
         self.clear_all_cache(parents=False, children=False)
-        if parent is not None:
-            parent.update_all_cache()
+        for p in parents:
+            p.update_all_cache()
+
+            for vfcti in vfctis:
```

Comment:
Any reader will need some help to decipher any variables named like this — let's have some human-readable name and reduce the cognitive load.